backend/apps/issue_categorization/models.py

```
@@ -0,0 +1,303 @@
+    └── DevOps
+        ├── CI/CD
+        └── Deployment
+    """
+    
+    CATEGORY_TYPES = [
+        ('project', 'Project'),
+        ('component', 'Component'),
+        ('sub_component', 'Sub-Component'),
+        ('feature', 'Feature'),
+        ('module', 'Module'),
+    ]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)